js/tests/ElggSpinnerTest.js

```diff
@@ -0,0 +1,68 @@
+define(function(require) {
+
+	var Spinner = require('elgg/Spinner');
+	var elgg = require('elgg');
+	
+	describe("elgg/Spinner", function() {
+	
+		it("start() creates the indicator", function() {
+			var spinner = new Spinner();
+
+			expect($('.elgg-spinner:visible').length).toBe(0);
+			spinner.start();
+			expect($('.elgg-spinner:visible').length).toBe(1);
+		});
+
+		it("stop() hides the indicator", function() {
+			var spinner = new Spinner({
+				fadeOpts: {
+					duration: 0,
+					complete: function () {
+						expect($('.elgg-spinner:visible').length).toBe(0);
+					}
+				}
+			});
+
+			spinner.start();
+			spinner.stop();
+		});
+
+		it("Indicator stays until all instances stopped", function() {
+			var s1 = new Spinner(),
+				s2 = new Spinner({
```

Comment:
Alternatively, just declare a new var on each line and let the minifier compress them (my preference).